casepro/msgs/views.py

```diff
@@ -186,7 +187,23 @@ def get_context_data(self, **kwargs):
             user = self.request.user
             page = int(self.request.GET.get('page', 1))
 
+            context['user_id'] = user.id
+
             search = self.derive_search()
+
+            # this is a refresh of messages
+            if self.request.GET.get('last_refresh', None):
+                new_messages = Message.search(org, user, search)
+
+                search['last_refresh'] = self.request.GET['last_refresh']
+                del search['after']
```

Comment:
Few questions:
  - Howcome `after` and `last_refresh` incompatible?
  - If `after` and `last_refresh` are incompatible, are `before` and `last_refresh` also perhaps incompatibility
  - Wondering if the logic of disregarding `after` param if `last_refresh` is given should be in `Message.search`?